app/policies/access_policy.rb

```diff
@@ -29,8 +29,15 @@ def configure
 
     # Any logged-in staff considered staff at present
     role :staff, proc { |user| !user.nil? } do
-      can :read, Kithe::Model # whether publisehd or not
-      can :update, Kithe::Model
+
+      can :read, Work
+      can :update, Work
+
+      can :read, Collection
+      can :update, Collection
+
+      can :read, Asset
+      can :update, Asset
```

Comment:
Oh here you are repeating these again... it's _supposed_ to work with the super-class... I'm wondering if it's breaking becuase of the thing I don't totally understand with access_granted where you can ask for a class instead of an instance?  If we change all those places you ask for a class to `can? :read, Kithe::Model` instead of `can?(:read, Asset) && can?(:read, Work)`.... maybe we can leave the policy in terms of just the single superclass? And I think it reads better anyway?

Reply:
It would read better but AccessGranted won't let us do that, as far as I can tell.